app/Http/Controllers/api/AuthController.php

```diff
@@ -15,20 +15,29 @@ public function login(Request $request)
             'dni' => 'required',
             'password' => 'required|string',
         ]);
+        $request->validate([
+            'dni' => 'required',
```

Comment:
I think this was a mistake from resolving conflicts. We need to delete this duplication. I leave it for next commit.